### ISSUE TITLE
Why can't I output qrcode?

### ISSUE THREAD

**Issue by sunrabbit123** [sdushantha/wifi-password#40]
![image](https://user-images.githubusercontent.com/64676070/107118947-9efb9300-68c7-11eb-8b3e-8fd8aa0a6fe5.png)

Why not I get qrcode image?

I get only Unknown String

What is ploblem?

my `OS` is `Windows 10`
my python version is `3.9.0`
my `wifi-password` version is `1.0.7`

**Comment by sdushantha** [comment]
I am unsure of why this is happening. Maybe this is happening because your system's language is Latin a language. I don't use Windows so I unfortunately can't debug this error.

**Comment by sunrabbit123** [comment]
My ststem language is korean

But this doesn't matter in code.

We need more examples.

**Comment by littlediobolic** [comment]
This is likely affecting all windows boxes, at least it does on mine aswell. It likely has to do with the symbols the qrcode library is sending. I dont believe the windows terminal is UTF-8, therefore UTF-8 specific symbols dont display properly like they do on Mac and Linux. However in testing this I also noticed that outputting to image might also be broken? At least it is for me.

@sdushantha I think this might be an issue of the `qrcode.print_tty()` function. The next best thing in Windows might be their `print_ascii` function but I havent tested it

Confirmation of it being broken on mine:
![wifi-password](https://user-images.githubusercontent.com/1341157/107455148-525cc580-6b1c-11eb-9cca-959b25d5d617.png)

Error when attempting to output to image file:
![image-error](https://user-images.githubusercontent.com/1341157/107455172-60aae180-6b1c-11eb-82b6-c99617da9858.png)

**Comment by sdushantha** [comment]
@sunrabbit123 @littlediobolic I think I found a solution to this issue.

In the beginning of the `main` function, could you add this code:

```python
import colorama
colorama.init()
```

This will emulate support for ANSI escape sequences, which is needed in order to display the QR code.

If you don't have `colorama` installed, you'll have to install it through `pip`:
```
python3 -m pip install --user colorama
```

**Comment by sunrabbit123** [comment]
Your solution works very sexy.

However, it would be better to write the phrase in generate_qr_code rather than main.

ex:
```python
def generate_qr_code(ssid, password, image=False):
    # Source: https://git.io/JtLIv
    text = f"WIFI:T:WPA;S:{ssid};P:{password};;"

    qr = qrcode.QRCode(version=2,
                       error_correction=qrcode.constants.ERROR_CORRECT_L,
                       box_size=10,
                       border=4)
    qr.add_data(text)

    if image:
        file_name = ssid.replace(" ", "_") + ".png"
        img = qr.make_image()
        img.save(file_name)
        print(f"QR code has been saved to {file_name}")
        os.system(file_name)
    else:
        import colorama
        colorama.init()
        qr.make()
        qr.print_tty()
```

**Comment by sdushantha** [comment]
This issue has been fixed in a4cbf0d
